core/sr-primitives/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 num-traits = { version = "0.2", default-features = false }
 integer-sqrt = { version = "0.1.2" }
 serde = { version = "1.0", optional = true, features = ["derive"] }
-parity-codec = { version = "3.3", default-features = false, features = ["derive"] }
+codec = { package = "parity-codec", version = "3.5.1", default-features = false, features = ["derive"] }
```

Comment:
This needs to be bumped to `4.0.0`, but there are dependencies that may prevent this.